internal/cli/config.go

```diff
@@ -165,8 +192,10 @@ func (pc PartitionConfig) Validate() error {
 	if len(pc.TestFilePaths) == 0 {
 		return errors.NewConfigurationError(
 			"Missing test file paths",
-			"No test file paths are given.",
-			"Please specify the path or paths to your test files as arguments to the 'captain partition' command.\n\n"+
+			"No test file paths are provided.\n",
+			"When using the run command, please specify the path or paths to your test files using the --partition-glob flag. "+
+				"You may specify this flag multiple times if needed.\n\n"+
+				"When using the partition command, please specify the path or paths to your test files as arguments.\n\n"+
 				"\tcaptain partition [flags] <filepath>\n\n"+
```

Comment:
Some of these configuration errors are pulling double duty as a result of partition config being used by `run` and `partition`